lib/card/index.js

```diff
@@ -35,10 +36,7 @@ class Card extends React.Component {
 
     const iconProps = {
       onClick: this.toggleCard,
-      className: classnames(
-        theme['more-icon'],
-        { [theme['less-icon']]: expanded },
-      ),
+      className: classnames(theme['expand-icon'], expanded ? theme.close : theme.open),
```

Comment:
```suggestion
      className: classnames(theme['expand-icon'], expanded ? theme.open : theme.close),
```
Expand Button UX feels reverse, Please check screenshot.
When collapsed Arrow points Upwards
![screenshot from 2019-01-01 12-10-55](https://user-images.githubusercontent.com/22497932/50570783-80f50780-0dbe-11e9-977d-74dcf2f3acc7.png)
And downwards when expanded
![screenshot from 2019-01-01 12-11-08](https://user-images.githubusercontent.com/22497932/50570784-85b9bb80-0dbe-11e9-924c-eb13fd2a1ac3.png)